server/db/models/order.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var Sequelize = require('sequelize');
+
+module.exports = function (db) { 
+	db.define('order', {
+		address: {
```

Comment:
might be easier for us in the long run to have addresses split into the following: street, street2, city, state, zip.